### ISSUE TITLE
ERROR: MacMenu: 'MFractor.ClearNugetPackagesCommand' maps to null command

### ISSUE THREAD

**Issue by KirillOsenkov** [mfractor/mfractor-feedback#29]
From the logs of VSMac 7.6.5.20:

```
ERROR [2018-09-16 22:46:14Z]: MacMenu: 'MFractor.ClearNugetPackagesCommand' maps to null command
```

Unfortunately I don't have any more information than this, just saw this randomly when reading a customer's log.

**Comment by matthewrdev** [comment]
Thanks for reporting this @KirillOsenkov. I'll look into this.

**Comment by matthewrdev** [comment]
This has been fixed and will be released in MFractor v3.6.
